docs/_posts/2014-08-31-sor.markdown

```diff
@@ -451,6 +451,51 @@ EmoDB allows for tagging updates (including deletes) with up to 3 event tags of
 generated by the updates. Databus listeners may subscribe to these event tags.
 Please note that these tags are only available on the corresponding databus events generated and are not a part of the document.
 
+### Stream update
+
+This is a way to stream a file full of changes to emo.
+
+What you need
+
+*   A file with all the updates. Let's call it updates.log
+    ```
+        [
+         {
+            "table": "review:testcustomer",
+            "key": "demo1",
+            "delta": "{\"field1\":\"value1\",\"field2\":\"value2\",\"field3\":\"value3\"}",
+            "consistency": "STRONG",
+            "audit": {
+              "comment": "Adding demo1 document"
+            }
+          },
+          {
+            "table": "review:testcustomer",
+            "key": "demo1",
+            "delta": "{\"field1\":\"value1.1\",\"field5\":\"value5\",\"field32\":\"value33\"}",
+            "consistency": "STRONG",
+            "audit": {
+              "comment": "Update and Add"
+            }
+          },
+          {
+            "table": "review:testcustomer:2",
+            "key": "demo1",
+            "delta": "{\"owner\":\"emodb-dev@bazaarvoice.com\",\"description\":\"Data Team QA\",\"issued\":\"2015-04-20T18:18:18.425+0000\",\"roles\":[\"data-team-qa\"]}",
```

Comment:
Can we take this email address out of the example?  Don't want to get spammed. ;)